frontend/src/pages/article/shared-article-page.tsx

```diff
@@ -26,6 +24,19 @@ const SharedArticlePage: FC = () => {
     }
   }, [dispatch, token]);
 
+  const { article, dataStatus } = useAppSelector(({ articles }) => ({
+    article: articles.article,
+    dataStatus: articles.dataStatus,
+  }));
+
+  const isLoading = !(
+    dataStatus === DataStatus.FULFILLED || dataStatus == DataStatus.REJECTED
+  );
```

Comment:
Maybe it'd be clearer to just use `isLoading = dataStatus === DataStatus.PENDING`? `